execution/executor-benchmark/src/lib.rs

```diff
@@ -599,7 +624,7 @@ mod tests {
 
     #[test]
     fn test_benchmark_transaction() {
-        test_generic_benchmark::<AptosVM>(Some(TransactionTypeArg::TokenV2AmbassadorMint), true);
+        test_generic_benchmark::<AptosVM>(Some(TransactionTypeArg::VectorPicture30k), true);
```

Comment:
Are you replacing the previous one? Why not run both?

Reply:
reverted. I use this for testing

might add separately a test for all types or something like that